### ISSUE TITLE
MHGen n3ds NAND Dump Size - Different to Listed values

### ISSUE THREAD

**Issue by idavies007** [Plailect/Guide#247]
I made multiple NAND backups on my MHGen N3DS, and they're all 1.82 GB (1,954,545,664 bytes), people in the #3dshacks IRC say they've seen other people reporting the same.

**Comment by MasterCheese** [comment]
[14:59:33] <Cygnus> Hi guys, I'm in the process of installing a9lh cfw for my brother's n3ds and succesfully downgraded to 9.2.0. When making a sysNAND backup, the size is always 1.82GB. The n3ds is part of the new Monster Hunter Generations bundle, could something be different with this model?
[15:00:50] <gruetzkopf> how are you checking?
[15:00:53] <gruetzkopf> check count in exact bytes
[15:02:19] <Cygnus> I did, it's smaller than it should be. I've also checked the backup of my own n3ds and that backup's size is exactly correct.

Logs from a few days ago of someone else reporting the same
